lib/sles4sap/microsoft_sdaf.pm

```diff
@@ -0,0 +1,177 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+#
+# Library used for Microsoft SDAF deployment
+
+package sles4sap::microsoft_sdaf;
+
+use strict;
+use warnings;
+use testapi;
+use Exporter qw(import);
+use Carp qw(croak);
+use mmapi qw(get_current_job_id);
+use utils qw(write_sut_file file_content_replace);
```

Comment:
`file_content_replace` seems not used

Reply:
I removed the sample test to avoid confusion. Keeping only relevant library for redirection.